src/helper/index.js

```diff
@@ -19,4 +22,14 @@ const sortByDiscount = (prevProd, minDiscount) => {
   return prevProd.filter((product) => product.discount >= minDiscount);
 };
 
-export { sortByCategory, sortByPrice, sortByDiscount };
+function sortByHL(prevProd, sortBy) {
+  if (sortBy && sortBy === "HIGH_TO_LOW") {
+    return [...prevProd.sort((a, b) => b.price - a.price)];
+  }
+  if (sortBy && sortBy === "LOW_TO_HIGH") {
+    return [...prevProd.sort((a, b) => a.price - b.price)];
+  }
+  return prevProd;
+}
+
+export { sortByCategory, sortByPrice, sortByDiscount, sortByHL };
```

Comment:
u can also make a different file for differnt helpers function to look the code cleaner